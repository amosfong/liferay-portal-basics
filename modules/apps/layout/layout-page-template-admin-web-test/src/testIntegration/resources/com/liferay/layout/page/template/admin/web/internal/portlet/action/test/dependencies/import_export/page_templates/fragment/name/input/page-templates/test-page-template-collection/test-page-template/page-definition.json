{
	"pageElement": {
		"id": "item1",
		"pageElements": [
			{
				"definition": {
					"fragment": {
						"key": "test-text-fragment"
					},
					"fragmentConfig": {
					},
					"fragmentFields": [
						{
							"id": "element-text",
							"value": {
								"fragmentLink": {
								},
								"text": {
									"value_i18n": {
										"en_US": "p2"
									}
								}
							}
						}
					],
					"name": "customName"
				},
				"id": "item2",
				"type": "Fragment"
			}
		],
		"type": "Root"
	},
	"settings": {
		"colorSchemeName": "01",
		"themeName": "Classic"
	},
	"version": 1.1
}