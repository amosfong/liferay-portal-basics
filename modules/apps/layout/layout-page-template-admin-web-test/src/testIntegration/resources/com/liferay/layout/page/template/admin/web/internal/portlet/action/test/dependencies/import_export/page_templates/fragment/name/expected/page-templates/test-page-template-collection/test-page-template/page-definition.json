{
	"pageElement": {
		"id": "item1",
		"pageElements": [
			{
				"definition": {
					"fragment": {
						"key": "test-text-fragment",
						"siteKey": "Â£{SITE_KEY}"
					},
					"fragmentConfig": {
					},
					"fragmentFields": [
						{
							"id": "element-text",
							"value": {
								"fragmentLink": {
								},
								"text": {
									"value_i18n": {
										"en_US": "p2"
									}
								}
							}
						}
					],
					"indexed": true,
					"name": "customName"
				},
				"id": "item2",
				"type": "Fragment"
			}
		],
		"type": "Root"
	},
	"settings": {
		"colorSchemeName": "01",
		"themeName": "Classic"
	},
	"version": 1.1
}